core/src/errors.rs

```diff
@@ -196,6 +196,9 @@ pub enum BridgeError {
 
     #[error("ERROR: {0}")]
     Error(String),
+
+    #[error("TonicError: {0}")]
```

Comment:
I suggest changing this into something like "RPC endpoint returned an error: {0}". Because users don't know Tonic is related to our RPC endpoints.

Reply:
Temporarily fixed in https://github.com/chainwayxyz/clementine/pull/361/commits/f9b1dc3b0062c9ade0092f4305e5ff18b5622035. But these errors should be more verbose with different error types.